setup.py

```diff
@@ -918,5 +918,11 @@ def run(self):  # noqa C901
             is_dynamic_lib=True,
             dependent_cmake_flags=["EXECUTORCH_BUILD_KERNELS_LLM_AOT"],
         ),
+        BuiltFile(
+            src_dir="backends/cuda/runtime/",
+            src_name="aoti_cuda_shims.lib",
+            dst="executorch/data/bin/",
```

Comment:
should it match the src_dir?

Reply:
I think this is probably clearer, though if theres an executorch/data/lib thatd probably be even better let me go check